app/helpers/development/dashboard_helper.rb

```diff
@@ -0,0 +1,59 @@
+module Development::DashboardHelper
+  def starred_proposal_discussion
```

Comment:
Too much DSL? I dunno, I found this to be a nice easy way to make a bunch of discussions I needed.